### PR TITLE
Number_Conversion(jainaman224#964)

### LINKED ISSUE #964: Number Conversion (Decimal To Binary)
can i start working on it ?

### PR DESCRIPTION
**Pull request by Manas2520**
Fixes #964 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:

Added a new file to convert numbers from decimal to binary form

#### Languages Used:

C++

#### Files Added:

Decimal_To_Binary.cpp

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Number_Conversion/C++/Decimal_To_Binary.cpp [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  cout<<"\nThe binary form is : ";
+  for(j=arlength-1; j>=0; j--)
+    cout<<b[j];
+}
+
+
+int main()
+{
+  cout<<"Enter the number you wish to convert:"<<endl;
```
Comment (raghav-dalmia) [inline Number_Conversion/C++/Decimal_To_Binary.cpp]: use 4 space indentation

File: Number_Conversion/C++/Decimal_To_Binary.cpp [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  int n;
+  cout<<"\n";
```
Comment (raghav-dalmia) [inline Number_Conversion/C++/Decimal_To_Binary.cpp]: remove this line

File: Number_Conversion/C++/Decimal_To_Binary.cpp [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  cin>>n;
+  printBinary(n);
+  cout<<"\n"; // Calling function printBinary to convert the number to binary form
+  return 0;
+}
+
+/*
+
```
Comment (raghav-dalmia) [inline Number_Conversion/C++/Decimal_To_Binary.cpp]: remove extra lines

**Comment by raghav-dalmia** [comment]
Squash your commits.

**Comment by raghav-dalmia** [comment]
Are you working on it?

**Comment by Manas2520** [comment]
My sincerest apologies sir. Something kept me preoccupied for the past few days but I will squash the commits and submit it very soon.Once again I am sorry for the delay.

**Comment by Manas2520** [comment]
Sir,please review my PR

File: Number_Conversion/C++/Decimal_To_Binary.cpp [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+#include <iostream>
+#include <math.h>
+using namespace std;
+
+// Function to convert the number from decimal to binary form
+
+void printBinary(int n){
+
+    int arlength=log2(n)+1; // command to find out number of bits in the given number which decides the size of the array
```
Comment (singh-shreya6) [inline Number_Conversion/C++/Decimal_To_Binary.cpp:11]: Please add space around operators like = +

File: Number_Conversion/C++/Decimal_To_Binary.cpp [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+    int b[arlength], j=0; // Array to store the binary number
```
Comment (singh-shreya6) [inline Number_Conversion/C++/Decimal_To_Binary.cpp:13]: Please add space around =

File: Number_Conversion/C++/Decimal_To_Binary.cpp [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    cout<<"\nThe binary form is : ";
+    for(j=arlength-1; j>=0; j--)
+        cout<<b[j];
+}
+
+int main()
+{
+    cout<<"Enter the number you wish to convert:"<<endl;
```
Comment (singh-shreya6) [inline Number_Conversion/C++/Decimal_To_Binary.cpp:28]: Please add space around << and >>

File: Number_Conversion/C++/Decimal_To_Binary.cpp [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    int b[size], j = 0; // Array to store the binary number
+
+    while(n! = 0){
+        b[j] = n % 2;
+        n = n / 2;
+        j++;
+    }
+
+    cout<<"\nThe binary form is : ";
+    for(j = size-1; j >= 0; j--)
```
Comment (singh-shreya6) [inline Number_Conversion/C++/Decimal_To_Binary.cpp:22]: Space around  -

File: Number_Conversion/C++/Decimal_To_Binary.cpp [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+#include <iostream>
+#include <math.h>
+using namespace std;
+
+// Function to convert the number from decimal to binary form
+
+void printBinary(int n){
+
+    int size = log2(n) + 1; // command to find out number of bits in the given number which decides the size of the array
```
Comment (singh-shreya6) [inline Number_Conversion/C++/Decimal_To_Binary.cpp:11]: Split the comment into 2 lines.

**Comment by Manas2520** [comment]
Ma'am,can I PLEASE make a new PR? I made a few grave mistakes ( please forgive me,I am completely new to this open source world) and I am facing a lot of trouble now.

**Comment by singh-shreya6** [comment]
sure make a new one. Closing this one.

**Pull request closed without merging**